File: Other OJs/SPOJ/INTEST.c
#include <stdio.h>
#include <stdint.h>

int read()
{
    int n = 0;
    char c;

    while ((c = getchar()) && (c != '\n' && c != -1 && c != ' ')) {
        n *= 10;
        n += c - '0';
    }

    return n;
}

int32_t main()
{
    int n, k, t, count = 0;

    n = read() + 1;
    k = read();
    
    while (--n) {
        t = read();

        count += !(t % k);
    }

    printf("%d\n", count);

    return 0;
}

File: Other OJs/SPOJ/INTEST.py
import io, os

input = io.BytesIO(os.read(0, os.fstat(0).st_size)).readline

n, k = map(int, input().split())

count = 0

for _ in range(n):
    t = int(input())
    
    count += (t % k == 0)
    
print(count)